### ISSUE TITLE
Scotchbox 3.0 install issues

### ISSUE THREAD

**Issue by ShaneShipston** [flurinduerst/WPDistillery#53]
It looks like they downgraded WP CLI in the newest scotchbox so the update command now fails causing nothing to work.

**Comment by flurinduerst** [comment]
Scotch Box 3.0 comes with WP-CLI 0.20.3. As far as I know, that's the same version that Scotch Box 2.5 had.

Anyway, I was able to replicate this issue and got the same WP-CLI Error using Scotch Box 3.0.

```
[...]
==> default: Running provisioner: shell...
    default: Running: /var/folders/k6/2kgbtz5n0pn_mldlnql1y0s00000gs/T/vagrant-shell20170621-1723-15f3dyg.sh
==> default: == Update WP CLI ==
==> default: Warning: The `wp cli update` command has an invalid synopsis part: [--patch]
==> default: Warning: The `wp cli update` command has an invalid synopsis part: [--minor]
==> default: Warning: The `wp cli update` command has an invalid synopsis part: [--nightly]
==> default: Warning: The `wp cli update` command has an invalid synopsis part: [--yes]
==> default: Error: Parameter errors:
==> default:  unknown --yes parameter
```

I'll see what we can do about this. Thanks for your report.
High priority!

**Comment by flurinduerst** [comment]
Got an update regarding this issue. The Problem is that the wpcli version on Scotch Box 3 doesn't support PHP 7.

* Scotch Box 3.0 comes with `php 7.0.19 ` and `wp-cli 0.20.3`
* WP-CLI support for `PHP7` came with `wp-cli 0.22.0`

That's why we cant update wp-cli on Scotch Box 3.0
Updating the pre-installed version of wpcli on Scotch Box to the newest version, or at least `0.23.0` (https://github.com/wp-cli/wp-cli/issues/3465) should fix this.

I'm in contact with @scotch-io to resolve this as soon as possible.

related:
* https://github.com/wp-cli/wp-cli/issues/1842
* https://github.com/wp-cli/wp-cli/issues/3465
* https://github.com/wp-cli/wp-cli/pull/2330

**Comment by ShaneShipston** [comment]
You can most likely get away with just setting `config.vm.box_version = "2.5"` within the vagrant file for now

**Comment by flurinduerst** [comment]
I just released Version 2.1 that now fully supports Scotch Box 3.0.

@ShaneShipston that would also be possible, thanks. But isn't it way cooler to just update WPDistillery to Scotch Box 3.0? 💪  
Thanks for your report, so we're back on track 14hours after you created the issue 😄
